### ISSUE TITLE
hi   the pytorch version of wing loss

### ISSUE THREAD

**Issue by shz1314** [TropComplique/wing-loss#3]
hi,
i have read the paper of CVPR 2018 about wing loss. I want to introduce the loss in my work. But my deblur work is based on the pytorch and i can transfor your tensorflow code about wing-loss into pytorch. Could you help me? Very thanks to you.
Song.

**Comment by shz1314** [comment]
Sorry , is  can not transfor
